packages/chaire-lib-common/src/tasks/dataImport/enhanceAndSaveOsmPolygonData.ts

```diff
@@ -152,11 +152,14 @@ export default class enhanceAndSaveOsmPolygonData extends GenericDataImportTask
             }
             if (feature.properties.building || feature.properties['building:part']) {
                 feature.properties['building:area'] = areaSqMeters;
-                // set default building:levels to 1:
-                feature.properties['building:levels:imputed'] = !!feature.properties['building:levels']; // TODO, check if there could be building:levels = 0 for small buildings and deal with it if so
-                feature.properties['building:levels'] = feature.properties['building:levels:imputed']
-                    ? 1
-                    : feature.properties['building:levels'];
+                // set building:levels to 1 if they do not already have one
+                // TODO, check if there could be building:levels = 0 for small buildings and deal with it if so
+                if (feature.properties['building:levels']) {
+                    feature.properties['building:levels:imputed'] = false;
+                } else {
+                    feature.properties['building:levels:imputed'] = true;
+                    feature.properties['building:levels'] = 1;
```

Comment:
Can you add a //TODO here : impute number of level based on height if available. (Maybe you have a task number for that)